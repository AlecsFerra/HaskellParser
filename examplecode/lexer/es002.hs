z =
  let
    x = let
      y = 3
    in y
  in x
