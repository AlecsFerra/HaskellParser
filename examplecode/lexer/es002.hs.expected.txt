[ OpenBrace
, LowerName "z"
, Equal
, Let
, OpenBrace
, LowerName "x"
, Equal
, Let
, OpenBrace
, LowerName "y"
, Equal
, IntLit 3
, CloseBrace
, Semicolon
, In
, LowerName "y"
, CloseBrace
, In
, LowerName "x"
, CloseBrace
]
